### PR TITLE
Several fixes and performance improvement

### LINKED ISSUE #60: Inefficient count-min sketch table representation
Table representation for count-min sketch uses python list. Using numpy would be more efficient. 
Should I make pull request with this?

### LINKED ISSUE #61: Missing method to aggregate count-min sketches
Count-min sketch has in theory property that 2 tables can be summed together which allows parallel count-min sketch building, but I don't see it implemented there.
Should I make pull request which implements it?

### PR DESCRIPTION
**Pull request by racinmat**
Fixes #60 
Fixes #61 
Fixes part of #62 .
Uses correct seed for default hash. 
Modified tests to reflect these changes.
Changed order of arguments for assertion to follow assert(expected, actual) paradigm. When not followed, unittest yields misleading error messages.
The problem with propagating collisions to larger depth still needs to be addressed though. 
Should I do it in this PR, or in some other?

### CODE REVIEW

**Comment by barrust** [comment]
@racinmat thank you for this PR. Generally, I like to keep pull requests to a single issue where possible to make it so that each part can be reviewed independently. I am going to keep this open for now and move discussion points into the issues that you submitted. 

Thanks!

**Comment by racinmat** [comment]
Regarding #61 I have checks for matching depth and width. Is there any reasonable way to assert that hashing functions are the same? They can be same in the sense of providing the same output for same input, but `cms._hash_function == cms2._hash_function` can be false for them, because they are not the same instance.

File: probables/countminsketch/countminsketch.py [hunk 1]
```diff
@@ -8,6 +8,9 @@
 import os
 from numbers import Number
 from struct import calcsize, pack, unpack
+from typing import Iterable
+
+import numpy as np
```
Comment (kapilt) [inline probables/countminsketch/countminsketch.py:13]: this is a big dependency to add

Comment (racinmat) [reply probables/countminsketch/countminsketch.py:13]: I know. I plan to perform and add benchmark demonstrating the efficiency so we can decide how much we gain using such big dependency.

**Comment by barrust** [comment]
closing as most has been added using separate pull requests

**Pull request closed without merging**